### PR TITLE
Moved Notion doc for creatingd repros into Docs

### PR DESCRIPTION
**Pull request by royanger**
A near word-for-word reproduction of the Notion doc on creating a reproduction. Updated the Next link to have a link to both the App and Pages Router quickstarts

### CODE REVIEW

**Comment by royanger** [comment]
@jescalan -- any concerns with moving this to Docs?

**Review by jescalan — APPROVED** [review]
Love it!

**Review by rachelnabors — APPROVED** [review]

File: docs/troubleshooting/create-a-minimal-reproduction.mdx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+Often times, the process of creating the minimal reproduction will clarify or resolve the original issue. This is great, if it does happen! That is the magic of minimal reproductions.
+
+## Submitting a minimal reproduction
+
+The Clerk support team **requires** minimal reproductions in order to start looking into reported bugs or issues for any users or customers that are not on an enterprise plan. 
+
+### Submission requirements
+
+Before submitting a minimal reproduction to the Clerk support team, ensure that the repository:
+- Is based off one of the starter templates above
```
Comment (royanger) [inline docs/troubleshooting/create-a-minimal-reproduction.mdx:83]: Suggestion: "Is based off one of the starter templates above or a very minimal recreation of the issue"

Comment (royanger) [reply docs/troubleshooting/create-a-minimal-reproduction.mdx:83]: Its not unreasonable for a customer to strip down their repo to a minimal reproduction. I think advanced customers will look over the steps and understand the goal and attempt to achieve it. We have definitely looked at reproductions that were not built from the ground up.

Comment (alexisintech) [reply docs/troubleshooting/create-a-minimal-reproduction.mdx]: Fair! Copy is updated :)

**Pull request merged**